Antonio_Ramos_Documentation

```diff
@@ -494,4 +436,83 @@ I am not installing arcpy because it appears (see line 449) that a previous chan
 
 - all data preparation now automated in python
 
-- need to schedule a zoom call to work on the issue with the k-fucntion local program not exporting a cluster file 
+- need to schedule a zoom call to work on the issue with the k-fucntion local program not exporting a cluster file
+
+8/30/2020
+
+- still troubleshooting lack of exported cluster file from "K-function 3Stage_Local.py" program. Will test out the follwoing ideas from Jake Carr.
+1. Double check that the lab coordinates are not too truncated, simimlar issue caused problems earlier on.
+
+2. make sure line 94 of the "K-function Local_3Stage.py" is working correctly. Meaning, it is correctly calculating the numnber of labs withitn the .5 1 5 and 10 mile radius of each lab. 
+
+9/2/2020
+- no issue found with the truncation of the xy coordinates
+
+- opening CA_Block_New in ArcMap to check the cluster counts for labs at .5 1 5 and 10 mile radius.
+
+- opening obsk data table in spyder variable expolorer to view if lab cluster counts match lab cluster countsI found in arcgis.
+
+- lab cluster counts do not match
+
+- will begin exploring how cluster counts are created
+
+- cluster counts are created by .kfunction() in kfunction_3stage
+
+- trying to understand how counts are created.
+
+9/6/2020
+- opening CA_Block_New in ArcMap to check the cluster counts for labs at .5 1 5 and 10 mile radius.
+
+- opening obsk data table in spyder variable expolorer to view if lab cluster counts match lab cluster countsI found in arcgis.
+
+- lab cluster counts do not match
+
+- will begin exploring how cluster counts are created
+
+- cluster counts are created by .kfunction() in kfunction_3stage
+
+- trying to understand how counts are created.
```

Comment:
This is a repeat of the lines immediately above

Reply:
I don't see any repeats on my end.

Reply:
I'm looking at lines 451-461 and comparing to lines 464-474